packages/ffe-form-react/src/HorizontalSelector.js

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { array, object, string, func } from 'prop-types';
+
+const HorizontalSelector = ({
+    options,
+    initialSelected,
+    label,
+    description,
+    onChange,
+    field,
+    ...rest
+}) => {
+    const [selectedValue, setSelectedValue] = useState(
+        Number(initialSelected.value),
+    );
+
+    const optionSelected = value => {
+        setSelectedValue(value);
+        onChange(field, value);
```

Comment:
vad er poengen med att senda med field in i komponenten? Allt den gjør er ju att brukas her.

Reply:
Kanske kunde ge mening hvis den var brukt som name på input fleterna så de blir unika.

Reply:
Dette er bare en konvensjon som var i bruk i vår app. Så det fjernet jeg. Den som bruker den kan fint ha tilgang til field uten at det ligger i denne komponenten.